android/BaseGameUtils/basegameutils/build.gradle

```diff
@@ -0,0 +1,34 @@
+apply plugin: 'com.android.library'
```

Comment:
why this module have build.gradle ? it is not there for other modules

Reply:
Some plugins require to be presented as libraries because they contain special source code and Android resources. Library projects contain build.gradle files.
Please note that we do not import library projects as project modules, instead we import this projects as AAR (Android archive) which contain necessary source code and resources. You can see this in android/config.json of this plugin This requires to rebuild .aar file after plugin source code has been modified with proces: Import project into Android Studio -> Add changes -> Build -> Rebuild project. This will rebuild .aar file, the path to which is already in config.json.

Reply:
Understood, Thank you.

Reply:
Yes, it is mandatory everytime after Java sources or Android resources are changed. It is related to library which is created and assembled as Android project in Android Studio. Libraries produce .aar instead of .apk. The building process is almost same as for apk files but is very quick. Then it requires to be committed and pushed to repository. Otherwise we can use CI and CD tools to keep aars accessible and to be built and tested automatically.

Reply:
okay, we can come back to it. But this should be noted down in README

Reply:
Done. Pull requests created for plugins:
Gameplay
Gamecircle
Facebook